spec/system/sign_in_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe 'Sign in' do
   let(:email_address) { user.email }
-  let(:password) { 'Str0ngPa$$w0rd' }
+  let(:password) { 'Str0ngPa$$w0rd12' }
```

Comment:
As with the `/ui` spec I think we should refer to the centralised config to obtain the value here, whilst using raw strings when we make assertions.

Reply:
This has been updated to use the centralised config value